RELEASENOTES.md

```diff
@@ -103,3 +103,7 @@
 * CB-7977 Mention `deviceready` in plugin docs
 * CB-7945 Made media.spec.15 and media.spec.16 auto tests green
 * CB-7700 cordova-plugin-media documentation translation: cordova-plugin-media
+
+### 0.2.16 (Jan 08, 2015)
+* CB-8252 **Android/Amazon**: Use message channel/PluginResult to send events from native -> Javascript. Removes all use of inline javascript/eval()s
```

Comment:
We update release notes as a part of releases. Going to not pull in the releasenotes commit.